sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2006,10 +1965,9 @@ def _from_generated(cls, span):
         )
 
     def __repr__(self):
-        return "DocumentSpan(offset={}, length={})".format(self.offset, self.length)
+        return f"DocumentSpan(offset={self.offset}, length={self.length})"
```

Comment:
huh seems pyupgrade had trouble f-stringifying the rest of the reprs in this file.

Reply:
Took me a while to come back to this.....but done!